business_requirement_project/wizard/generate_projects.py

```diff
@@ -104,6 +109,7 @@ def generate_br_projects(self, parent_project, br, project_ids, task_ids):
                 br_project_val = self._prepare_project_vals(
                     br, parent_project)
                 br_project = project_obj.create(br_project_val)
+                br.linked_project = br_project.id
```

Comment:
Shouldnot there be a condition similar to?
if not br.linked_project then
                br_project = project_obj.create(br_project_val)
                br.linked_project = br_project.id
else
                br_project = br.linked_project